src/main/ui/src/components/SearchByState/SearchByState.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
I noticed that on the app startup that the state alabama is selected in the dropdown, but all states show in the list. Please change the initial selection to `Choose a state` that way all the states showing makes sense. Also I see ZERO styling on this. Let's look at emily/candide's work and match it up please.

Reply:
Also make sure to remove all comments before submitting

Thank you!